easybuild/easyconfigs/m/monitor/monitor-1.1.2.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'ConfigureMake'
+
+name = 'monitor'
+version = '1.1.2'
+
+homepage = 'https://github.com/gjbex/monitor'
+description = "monitor logs cpu and memory usage of a running application"
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/gjbex/monitor/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['0e034526c8b6fd3880df5d65a6b2599b5e9fd88ea4506aa29bd6c7f016fb9753']
+
+osdependencies = ['perl']
```

Comment:
My failure is because the standard sanity check for Perl modules uses `perldoc`.

```
== 2022-01-03 18:41:28,202 extension.py:275 WARNING Sanity check for 'Getopt::Long' extension failed: command "perldoc -lm Getopt::Long " failed; output:
You need to install the perl-doc package to use this program.
```

In my Ubuntu install, `/usr/bin/perldoc` is:
```
#!/bin/sh
# place-holder, diverted by perl-doc
echo You need to install the perl-doc package to use this program. >&2
exit 1
```

So, I think that we need an extra OS dependency here to ensure that it is available.

Reply:
On CentOS 7 and RHEL 8, `perldoc` is provided by `perl-Pod-Perldoc`.

What should that be on Ubuntu, just `perl-doc`?

Reply:
An alternative here could be to skip the "import" check for the extensions, which requires `perldoc`, since we have a proper sanity check command that'll verify whether the required Perl packages are installed and can be loaded?

Reply:
Yes, it is `perl-doc` for Ubuntu.

I'd be fine with skipping the import checks where we have a suitable sanity check command.

Reply:
fixed in 9ccc68b